test/samples-js-extra/AddProductsNoPartialUpsert/index.js

```diff
@@ -0,0 +1,24 @@
+module.exports = async function (context) {
+    context.log('JavaScript HTTP trigger function processed a request.');
+
+    var products = [];
+    for(let i = 0; i < 1000; i++) {
+        products.add({"productId": i,
+        "name": "test"+i,
+        "cost": i});
+    }
+    const invalidProduct = {
+        "productid": 1000,
+        "name": null,
+        "cost": 1000
+    };
+
+    products.add(invalidProduct);
+
+    context.bindings.products = JSON.stringify(products);
```

Comment:
It might be worth it to note that this isn't really testing the same thing as the C# version since JS (and most of the other languages) don't support the ICollector style of streaming uploads (at least as far as I know)

This test is still fine to have for additional verification but it does have different behavior which should be called out.